package-lock.json

```diff
@@ -21,7 +21,7 @@
         "@typescript-eslint/parser": "^5.50.0",
         "caller": "^1.1.0",
         "debug": "^4.3.4",
-        "eslint": "8.39.0",
+        "eslint": "^8.30.0",
```

Comment:
Can you quickly sanity check that this still runs on a typescript file in the repo? There was an eslint bug that caused me to pint his beforehand.